src/origin.rs

```diff
@@ -95,22 +95,12 @@ pub(crate) fn apply_origin(
             &Origin,
             &mut OriginState,
             &Transform,
-            Changed<Transform>,
```

Comment:
This functionality can be replaced by using `Ref<Transform>` above and checking `transform.is_changed()` in the if statement below.